Mathlib/MeasureTheory/Integral/Lebesgue/Basic.lean

```diff
@@ -779,14 +827,16 @@ theorem lintegral_add_mul_meas_add_le_le_lintegral {f g : α → ℝ≥0∞} (hl
   simp only [indicator_apply]; split_ifs with hx₂
   exacts [hx₂, (add_zero _).trans_le <| (hφ_le x).trans hx₁]
 
-/-- **Markov's inequality** also known as **Chebyshev's first inequality**. -/
+/-- **Markov's inequality**, multiplication form for `AEMeasurable` functions. -/
```

Comment:
Please don't drop Chebyshev's name here. He was Markov's teacher and he proved it first (see Wikipedia).

Reply:
Done.